jiant/scripts/download_data/datasets/nlp_tasks.py

```diff
@@ -0,0 +1,129 @@
+        "path": "glue",
+        "name": "mnli",
+        "label_map": {0: "contradiction", 1: "entailment", 2: "neutral"},
+        "phase_map": {"validation_mismatched": "val", "test_mismatched": "test"},
+        "phase_list": ["val", "test"],
+        "jiant_task_name": "mnli",
+    },
+    "mrpc": {
+        "path": "glue",
+        "name": "mrpc",
+        "field_map": {"sentence1": "text_a", "sentence2": "text_b"},
+        "label_map": {0: "0", 1: "1"},
+    },
+    "qnli": {
+        "path": "glue",
+        "name": "qnli",
+        "field_map": {"question": "premise", "sentence": "hypothesis"},
+        "label_map": {0: "contradiction", 1: "entailment", 2: "neutral"},
+    },
+    "qqp": {
+        "path": "glue",
+        "name": "qqp",
+        "field_map": {"question1": "text_a", "question2": "text_b"},
+        "label_map": {0: "0", 1: "1"},
+    },
+    "rte": {
+        "path": "glue",
+        "name": "rte",
+        "field_map": {"sentence1": "premise", "sentence2": "hypothesis"},
+        "label_map": {0: "entailment", 1: "not_entailment"},
+    },
+    "sst": {
+        "path": "glue",
+        "name": "sst2",
+        "field_map": {"sentence": "text"},
+        "label_map": {0: "0", 1: "1"},
+    },
+    "stsb": {
+        "path": "glue",
+        "name": "stsb",
+        "field_map": {"sentence1": "text_a", "sentence2": "text_b"},
+    },
+    "wnli": {
+        "path": "glue",
+        "name": "wnli",
+        "field_map": {"sentence1": "premise", "sentence2": "hypothesis"},
+        "label_map": {0: "0", 1: "1"},
+    },
+    "glue_diagnostics": {
+        "path": "glue",
+        "name": "ax",
+        "phase_map": None,
+        "jiant_task_name": "mnli",
+    },
+    # === SuperGLUE === #
+    "boolq": {"path": "super_glue", "name": "boolq", "label_map": {0: False, 1: True}},
+    "cb": {
+        "path": "super_glue",
+        "name": "cb",
+        "label_map": {0: "contradiction", 1: "entailment", 2: "neutral"},
+    },
+    "copa": {"path": "super_glue", "name": "copa"},
+    "multirc": {"path": "super_glue", "name": "multirc"},
+    "record": {"path": "super_glue", "name": "record"},
+    "wic": {"path": "super_glue", "name": "wic", "label_map": {0: False, 1: True}},
+    "wsc": {"path": "super_glue", "name": "wsc.fixed", "label_map": {0: False, 1: True}},
+    "superglue_broadcoverage_diagnostics": {
+        "path": "super_glue",
+        "name": "axb",
+        "field_map": {"sentence1": "premise", "sentence2": "hypothesis"},
+        "label_map": {0: "entailment", 1: "not_entailment"},
+        "phase_map": None,
+        "jiant_task_name": "rte",
+    },
+    "superglue_winogender_diagnostics": {
+        "path": "super_glue",
+        "name": "axg",
+        "label_map": {0: "entailment", 1: "not_entailment"},
+        "phase_map": None,
+        "jiant_task_name": "rte",
+    },
+    # === Other === #
+    "snli": {"path": "snli", "label_map": {0: "contradiction", 1: "entailment", 2: "neutral"}},
```

Comment:
snli needs "name" and has an error

Reply:
Good point. Will switch to `.get('name')`.